pkg/deploy/deploy.go

```diff
@@ -86,14 +61,46 @@ func DeployConfigs(client dtclient.Client, apis api.APIs, sortedConfigs []config
 			if !opts.ContinueOnErr && !opts.DryRun {
 				return errors
 			}
-		} else {
-			entityMap.put(entity.Coordinate, entity)
+		} else if entity != nil {
```

Comment:
If the skip above is changed to what it was before, this will make more sense again.
If an error occurs, we should not care about the entity.

Reply:
For this case
```
		case config.EntityType:
			log.Debug("Entity are not deployable, skipping entity type: %s", t.EntitiesType)
			continue
```
there is no error, and nothing will be put in `entityMap`.